### ISSUE TITLE
"Page not found." error

### ISSUE THREAD

**Issue by aeiler** [sumpfork/dominiontabs#34]
I just got a new(to me) expansion and went to generate the next set of tabs and got a "Page not found." error message.

I just created a few sets of tabs in November without issue. My guess is the update on 12/1/2015 may have added a little bug.

I did a little more testing and have found this issue only appears when you check the 'Cropmarks instead of outlines' option.
Please let me know if you need any more information or screenshots. This error should be easily replicated though.

**Comment by wvoigt** [comment]
This is the error: draw.py, line 129, in drawOutline  self.dividerHeight, -cmw, self.dividerHeight)
AttributeError: 'DividerDrawer' object has no attribute 'dividerHeight'
And this is the code:
```
            if cropmarksleft or cropmarksright:
                self.canvas.line(-2 * cmw, 0, -cmw, 0)
                self.canvas.line(-2 * cmw,
                                 self.dividerHeight, -cmw, self.dividerHeight)
                if y > 0:
                    self.canvas.line(-2 * cmw,
                                     self.options.dividerHeight, -cmw, self.options.dividerHeight)
```
So self.dividerHeight does not exist.  Everywhere else this has been changed to self.options.xxxx.
But I'm not sure exactly what this the code on line 129 is doing. The part after "if y>0" has been changed, but line 129 is changed to self.options.dividerHeight, then I'm not sure what the "if y>0" is for, since the code is the same.  Might be a minus sign missing on one or the other code segment.

**Comment by sumpfork** [comment]
Finally got time to look at this - oops, yes, bad refactoring. Should be fixed with 5d58c4b.
